thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -140,7 +140,7 @@ export function setUpTimeRangeOptions(datesKeys, duration) {
    datesKeys.forEach((value) => {
      const currVal = dateKeyMap.get(value);
      const label = currVal[0];
-     const start = (label === 'Today') ? moment().startOf('day') : moment().subtract(currVal[1], currVal[2]).startOf('hour').utc();
+     const start = (label === 'Today') ? moment().startOf('day') : moment().subtract(currVal[1], currVal[2]).utc();
```

Comment:
I believe this was set to avoid unexpected anomalies on the edge of an hour, but probably not making a big difference.